confidant/routes/credentials.py

```diff
@@ -86,48 +86,49 @@ def get_credential_list():
     :statuscode 200: Success
     :statuscode 403: Client does not have permissions to list credentials.
     """
```

Comment:
it's interesting how git detects the diff: the only changes made is add `with stats.timer(` and added indent to following lines